spec/lib/jobs/install-os-job-spec.js

```diff
@@ -379,5 +379,28 @@ describe('Install OS Job', function () {
                    .to.throw('The uid should between 500 and 65535 (>=500, <=65535)');
         });
         
+        it('should throw error when mountPoint is not a string', function() {
+            job.options.installPartitions = [{ mountPoint:{}, size:'500', fsType:'ext3' }];
+            return expect(job._validateOptions.bind(job,{}))
+                   .to.throw('mountPoint must be a string');
+        });
+
+        it('should throw error when size is not a string', function() {
+            job.options.installPartitions = [{ mountPoint:'/boot', size:{}, fsType:'ext3' }];
```

Comment:
As my understanding, unit-test is to test a portion of code has expected behavior for every (as many as) possible input. The test coverage is just a kind of measurement, not the target. Even 100% coverage doesn't mean the unit-test is good enough to have taken every possible input into account.

Meanwhile, unit-test is to test the behavior on input, so it should not care the implementation detail. So in your case,  unit-test doesn't care you use `assert.string` or another assertion library or even you design the assertion by yourself. Unit-test just cares if I pass a non-string value, the validation should throw an error. Though we cannot list all non-string input, but at least we can test those common cases. If the `size` is from a calculation, maybe it ends up with `NaN`; if the `size` is unpurposely omitted, it is `undefined`;...

Anyway, this comment is not gating for me, just a recommendation to take as many as possible input into account .